src/getValidators.unit.test.ts

```diff
@@ -0,0 +1,531 @@
+import {
+  Address,
+  EIP1193RequestFn,
+  Hex,
+  createPublicClient,
+  createTransport,
+  encodeFunctionData,
+  http,
+} from 'viem';
+import { arbitrum } from 'viem/chains';
+import { it, expect, vi, describe } from 'vitest';
+import { getValidators } from './getValidators';
+import { rollupAdminLogicABI, safeL2ABI } from './abi';
+import { rollupAdminLogicPrepareFunctionData } from './rollupAdminLogicPrepareTransactionRequest';
+
+// Xai
+const rollupAdminLogicAddress = '0xc47dacfbaa80bd9d8112f4e8069482c2a3221336';
```

Comment:
the rollup & rollup admin logic is on the same address

it is confusing that there are two different addresses for `rollupAdminLogicAddress` and `rollupAddress` in this file

I assume that is because they are the rollup addresses of 2 different Orbit chains

can you rename them and give some comments to them?